src/main/java/org/jenkinsci/plugins/workflow/flow/StashManager.java

```diff
@@ -165,6 +167,30 @@ public static void copyAll(@Nonnull Run<?,?> from, @Nonnull Run<?,?> to) throws
         FileUtils.copyDirectory(fromStorage, storage(to));
     }
 
+    /**
+     * Get the filenames included in a particular stash.
```

Comment:
Should include a hard warning that this extracts the whole stash -- this is not a free or cheap operation.